packages/compartment-mapper/src/node-modules.js

```diff
@@ -611,11 +611,6 @@ const translateGraph = (
       },
       policy,
     );
-    // do not include compartments for packages not covered by policy
-    if (policy && !packagePolicy) {
-      // eslint-disable-next-line no-continue
-      continue;
-    }
```

Comment:
this was a premature optimization, we could introduce a strict mode in the future but for real applications with npm dependencies it doesn't make sense to generate  and maintain policy entries for powerless packages. 

The only alternative approach would be to process the policy and assume empty entries exist for all packages allowed for import. 

Looking at the positive outcomes from this change here, I'd rather avoid optimizing this.

Reply:
And [here](https://github.com/boneskull/endo/blob/ba77c6644962cf25510554fc9318d8a8094fd39a/packages/compartment-mapper/src/import-hook.js#L236)

Reply:
And [here](https://github.com/boneskull/endo/blob/ba77c6644962cf25510554fc9318d8a8094fd39a/packages/compartment-mapper/src/import-archive.js#L109) too 😄